litestar/openapi/controller.py

```diff
@@ -11,34 +11,19 @@
 from litestar.enums import MediaType, OpenAPIMediaType
 from litestar.exceptions import ImproperlyConfiguredException
 from litestar.handlers import get
-from litestar.response import Response
+from litestar.response.base import ASGIResponse
 from litestar.serialization import encode_json
 from litestar.status_codes import HTTP_404_NOT_FOUND
+from litestar.utils.helpers import get_enum_string_value
 
-__all__ = ("OpenAPIController", "OpenAPISchemaResponse")
+__all__ = ("OpenAPIController",)
 
 
 if TYPE_CHECKING:
     from litestar.connection.request import Request
     from litestar.openapi.spec.open_api import OpenAPI
 
-
-class OpenAPISchemaResponse(Response):
```

Comment:
This is removed as we return `ASGIResponse`s directly from the handlers on the controller and are passing encoded data directly to them.